src/main/java/com/exactpro/th2/readlog/RegexLogParser.java

```diff
@@ -13,56 +13,107 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 package com.exactpro.th2.readlog;
 
-import java.util.ArrayList;
-import java.util.Collections;
+import java.time.DateTimeException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
 import java.util.List;
+import java.util.Map;
+import java.util.Objects;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+import com.exactpro.th2.readlog.cfg.AliasConfiguration;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class RegexLogParser {
-	private static final Logger logger = LoggerFactory.getLogger(RegexLogParser.class);
-    private final Pattern pattern;
-	private final List<Integer> regexGroups;
+    private static final Logger logger = LoggerFactory.getLogger(RegexLogParser.class);
+    private final Map<String, AliasConfiguration> cfg;
+
+    public RegexLogParser(Map<String, AliasConfiguration> cfg) {
+        this.cfg = Objects.requireNonNull(cfg, "'Cfg' parameter");
+        if (cfg.isEmpty()) {
+            throw new IllegalArgumentException("At least one alis must be specified");
+        }
+    }
+
+    public LogData parse(String alias, String raw) {
```

Comment:
Why do we hide problems with configuration?
A user will know about the problem only from the log.